### DISCUSSION TITLE
Sign in form has both Vanjaro and Standard login tabs

### DISCUSSION THREAD

**Discussion by puresystems** [vanjarosoftware/Vanjaro.Platform#2041] [Q&A] [answered]
Hi, is there a way to only show the Vanjaro signin form (using the Sign In Form design block) as when we add it to a page and publish, it shows 2 tabs above the form, Vanjaro and Standard, the standard form doesn't work and is confusing.

**Comment by nikhilmandeeps** [comment, marked as answer]
Please Insert/Update in **PortalSettings** Table (**SettingName ='DNN_Enabled' and SettingValue='False'**)

**Reply by puresystems** [reply]
Thanks, that's solved the issue.

**Reply by eswanzey** [reply]
I'm trying to diagnose some odd login behavior on Vanjaro hybrid sites, and am comparing some values in the PortalSettings tables among hybrid and native Vanjaro sites.

Some sites have a SettingName='IsVanjaroInstalled' and SettingValue='-1', some sites don't have the setting at all. Shouldn't they all have the setting? Why is it '-1' on those that do??

One or two sites have SettingName='Vanjaro_Impersonation' but this setting is missing on the majority of sites I checked. What is this setting for? Shouldn't they all have this setting?
